src/pages/components/RenderList.tsx

```diff
@@ -1,10 +1,11 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 import React, { useState } from "react";
 import {
```

Comment:
![01⁄05⁄212021-01-05Tuesday 5_1](https://user-images.githubusercontent.com/1283920/103567806-778d6100-4eea-11eb-8c96-53d7de1e03bc.png)

Reply:
Deleted the Render List